docs/content/integrations/fivetran.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: "Using Dagster with Fivetran"
+description: Integrate your Fivetran connectors into Dagster.
+---
+
+# Using Fivetran with Dagster
+
+Dagster can orchestrate your Fivetran connectors, making it easy to chain an Fivetran sync with upstream or downstream steps in your workflow.
+
+This guide focuses on how to work with Fivetran connectors using Dagster's [software-defined asset (SDA)](/concepts/assets/software-defined-assets) framework.
+
+<br />
+
+<center>
+  <Image
+    alt="Screenshot of the Fivetran UI displaying two Fivetran connectors, and the Dagster UI displaying a graph of related assets."
+    src="/images/integrations/fivetran/fivetran_assets.png"
+    width={2301}
+    height={1232}
+  />
+</center>
+
+---
+
+## Fivetran connectors and Dagster software-defined assets
+
+An [Fivetran connector](https://fivetran.com/docs/getting-started/core-concepts) defines a series of schemas and tables which are synced between a source and a destination. During a sync, data is retrieved from the source and written to the destination as one or more tables. Dagster represents each of the replicas generated in the destination as a software-defined asset. This enables you to easily:
+
+- Visualize the schemas and tables involved in an Fivetran connector and execute a sync from Dagster
+- Define downstream computations which depend on replicas produced by Fivetran
+- Track historical metadata and logs for each table
+- Track data lineage through Fivetran and other tools
+
+---
+
+## Prerequisites
+
+To get started, you will need to install the `dagster` and `dagster-fivetran` Python packages:
+
+```bash
+pip install dagster dagster-fivetran
+```
+
+You'll also want to have set up [a Fivetran instance](https://fivetran.com/docs/getting-started) and have a Fivetran [API key and API secret](https://fivetran.com/docs/rest-api/getting-started), which can be generated by a Fivetran Account Administrator.
+
+---
+
+## Step 1: Connecting to Fivetran
+
+The first step in using Fivetran with Dagster is to tell Dagster how to connect to your Fivetran instance using an Fivetran [resource](/concepts/resources). This resource contains the credentials needed to access your Fivetran instance.
+
+We will supply our credentials as environment variables:
```

Comment:
Should we add a link to the new env vars + secrets guide in here somewhere? https://docs.dagster.io/guides/dagster/using-environment-variables-and-secrets

Reply:
Good idea, added 👍